inc/admin/ui/notices.php

```diff
@@ -490,7 +490,9 @@ function rocket_analytics_optin_notice() {
 		return;
 	}
 
-	if ( get_rocket_option( 'analytics_enabled' ) ) {
+	// The new option being maintained is rocket_mixpanel_optin.
+	// analytics_enabled is still kept and checked for backward compatibility.
+	if ( get_rocket_option( 'analytics_enabled' ) || get_option( 'rocket_mixpanel_optin' ) ) {
```

Comment:
That's not necessary, the old value was upgraded to the new option during the update routine when we switched to the mixpanel library